stress/shell/src/main/java/alluxio/stress/cli/GetPinnedFileIdsBench.java

```diff
@@ -163,12 +165,12 @@ private RpcTaskResult runRPC() throws Exception {
         pointStopwatch.reset();
         pointStopwatch.start();
 
-        Set<Long> pinned = client.getPinList();
+        int numPinnedFiles = client.getPinListLength();
 
         pointStopwatch.stop();
 
-        if (pinned.size() != mParameters.mNumFiles) {
-          result.addError(String.format("Unexpected number of files: %d", pinned.size()));
+        if (numPinnedFiles != mParameters.mNumFiles) {
```

Comment:
If any of your files failed to generate due to random string collision, this will be an error?

Reply:
Technically yes, but the probability of such a collision is negligible, even given very large number of files (>1B). We could abort the benchmark during `prepare()` in this case, but the gain will be also very marginal.